src/Ombi/ClientApp/src/app/my-nav/my-nav.component.html

```diff
@@ -3,7 +3,7 @@
         [attr.role]="(isHandset$ | async) ? 'dialog' : 'navigation'" [mode]="(isHandset$ | async) ? 'over' : 'side'"
         [opened]="!(isHandset$ | async)">
 
-        <mat-toolbar class="application-name" id="nav-applicationName">{{applicationName}}</mat-toolbar>
+        <mat-toolbar class="application-name" id="nav-applicationName"><a class="application-name-link" [href]="applicationUrl">{{applicationName}}</a></mat-toolbar>
```

Comment:
One thing I just thought about, `applicationUrl` could be null.

It might be worth applying the `<a href` when the applicationUrl is not null